js-foundations.js

```diff
@@ -2,27 +2,89 @@
 // Coding practical questions
 
 // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
+var number = [15, 0, 385, -7]
+var isDivisableByThree = function(number) {
+if (isNaN (number)) {
+return "Input was not a number.";
+}
+else if (number % 3 === 0) {
+return 'is divisble by 3'
+
+}
+else {
+return 'is not divisble by 3';
+
+}
+};
+//or ?
+function divisBy3(number){
+    console.log(`${number} is ${number % 3 === 0 ? '' : 'not'} divisible by 3`)
+}
 
 
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
+const helloMe = {
+    firstName: "Brenden",
+    lastName: "Barta",
+    age: 20,
+    height: "5'9",
+    describe(){ return(`${this.firstName} ${this.lastName} is a ${this.sign} years old and is ${this.height} .`)}
+}
 
+console.log(helloMe.describe())
 
 
 // ------------------------------------- Consider the variable:
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 
 // 3a. Write a function that returns every other item from the array as one string. Expected output: "streetlamp teeth nephew database"
+var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 
+console.log(randomNouns.join());
 
+//or
+
+function func() { 
+  var randomNouns = [ 1, 2, 3, 4, 5, 6 ]; 
+  document.write(randomNouns.join()); 
+}  
+  
+func(); 
 
-// 3b. Create a function that takes in the variable and returns the array with all the words capitalized. Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+// 3b. Create a function that takes in the variable and returns the array with all the words capitalized. Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
+var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
+function uppercase(str)
+{
+  var array1 = str.split(' ');
+  var newarray1 = [];
+    
+  for(var x = 0; x < array1.length; x++){
+      newarray1.push(array1[x].charAt(0).toUpperCase()+array1[x].slice(1));
+  }
+  return newarray1.join(' ');
+}
+console.log(uppercase(randomNouns));
 
```

Comment:
Looks good! You actually don't need the str.split(" ") because you are bringing in an array as your argument. If you get rid of that line and change the array1 to just str from your argument, this code will work.